### PR TITLE
Rianfowler/sc 70719/cleanup various parts of custom branding make success warn failure themeable

### PR DESCRIPTION
**Pull request by rianfowler**
#### What this PR does / why we need it:
- make the success fail and warning colors themeable 

#### Which issue(s) this PR fixes:

Fixes # https://app.shortcut.com/replicated/story/70719/cleanup-various-parts-of-custom-branding

## Steps to reproduce

use this as the style config: 
Screenshot 2023-03-07 at 16 48 54
Screenshot 2023-03-07 at 16 37 39
Screenshot 2023-03-08 at 09 34 06

```
:root {
  --card-color: #e9f3ff;
  --error-color: #f2ff49;
  --error-color-light: #ff4242;
  --success-color: #b3fffc;
  --success-color-light: #645dd7;
  --warning-color: #fb62f6;
  --warning-color-light: #b3fffc;
  --file-tree-color: #a3bbb6;
  --file-tree-highlight-color: #93a9a4;
  --icon-color: #000000;
  --link-color: #93a9a4;
  --primary-color: #a3bbb6;
  --primary-disabled-color: #bfc8d5;
  --primary-hover-color: #93a9a4;
  --secondary-hover-color: #e9f3ff;
  --subnav-color: #e9f3ff; 
  --tooltip-color: #5b5b5b;
}
```

#### Does this PR introduce a user-facing change?

```release-note
none
```

#### Does this PR require documentation?

none

### CODE REVIEW

File: web/src/scss/variables.scss [hunk 1]
```diff
@@ -47,6 +49,12 @@ $border-radius: 4px;
   --primary-disabled-color: #{$primary-less-light-color};
   --primary-hover-color: #{$primary-dark-color};
   --secondary-hover-color: #{$primary-light-color};
+  --error-color: #{$error-color};
+  --error-color-light: #{$error-color-light};
+  --warning-color: #{$warning-color};
+  --warning-color-light: #{$warning-color-light};
+  --success-color: #{$success-color};
+  --success-color-light: #{$success-color-light};
```
Comment (rianfowler) [inline web/src/scss/variables.scss:57]: this PR swaps the scss versions of these variables for the css variables.

File: web/src/features/Dashboard/components/DashboardVersionCard.tsx [hunk 2]
```diff
@@ -1335,6 +1335,7 @@ const DashboardVersionCard = (props: Props) => {
       return (
         <div className="error-block-wrapper u-marginTop--20 u-marginBottom--10 flex flex1">
           <span className="u-textColor--error">
+            error
```
Comment (erquhart) [inline web/src/features/Dashboard/components/DashboardVersionCard.tsx:1338]: Intentional?

**Review by erquhart — DISMISSED** [review]

File: web/src/features/Dashboard/components/DashboardVersionCard.tsx (hunk 2 above)
Comment (rianfowler) [reply web/src/features/Dashboard/components/DashboardVersionCard.tsx:1338]: nope good catch

Comment (rianfowler) [reply web/src/features/Dashboard/components/DashboardVersionCard.tsx:1338]:
```suggestion
```

**Review by erquhart — APPROVED** [review]

**Pull request merged**